controllers/feeds/getFeedChat.js

```diff
@@ -124,6 +125,8 @@ const finalize = (req, id, myReaction, newGroup, activity_id, constantActor) =>
     }
 
     if (myReaction.kind === 'comment') {
+      const user = karmaScores.find((user) => user.user_id === myReaction.user_id);
+      myReaction.user.karmaScores = roundingKarmaScore(user?.karma_score || 0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic here fetches the user from the `karmaScores` array and updates the `user.karmaScores` property with the rounded karma score. This is a good approach as it ensures that the karma score is always up-to-date and rounded off before being used.
```diff
+ const user = karmaScores.find((user) => user.user_id === myReaction.user_id);
+ myReaction.user.karmaScores = roundingKarmaScore(user?.karma_score || 0);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->